mentors/julia/w.json

```
@@ -1 +1,10 @@
-[]
+[
+{
+  "github_username": "Wikunia",
+  "name": "Ole Kröger",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "I'm a master student in computer science in Germany and love to work on small and increasingly big personal open source projects in Julia."
+}
```

```suggestion
  {
    "github_username": "Wikunia",
    "name": "Ole Kröger",
    "link_text": null,
    "link_url": null,
    "avatar_url": null,
    "bio": "I'm a master student in computer science in Germany and love to work on small and increasingly big personal open source projects in Julia."
  }
```
Content looks fine, just needs some minor indentation adjustment.

---

Thanks I copy pasted it from the readme and hoped it works :D

---

Yes, I see what you did now that you mention it.  You did right 😃 the readme should probably be updated so that all the lines are indented two more spaces.  Only the `[` and `]` characters are not indented and they are not included in the example which I agree that they not be included.